api/bases/dataplane.openstack.org_openstackdataplanenodesets.yaml

```diff
@@ -197,6 +197,29 @@ spec:
                 - ctlplaneInterface
                 - deploymentSSHSecret
                 type: object
+              containerImages:
+                properties:
+                  frr:
+                    type: string
+                  iscsiD:
+                    type: string
+                  logrotate:
```

Comment:
as an aside logroate should hopefully go away when we complete the move to having all services log to the journal